documentation/book/cluster-operator.adoc

```diff
@@ -1060,111 +1060,121 @@ They will not be added to the Docker image from a previous build.
+
+KafkaConnect and KafkaConnectS2I pods use the default ServiceAccount, since they require no access to {ProductPlatformName} resources.
 
-For the operator to function it needs permission within the {ProductPlatformName} cluster to interact with
-the resources it manages (ConfigMaps, Pods, Deployments, StatefulSets, Services and so on).
-Such permission is described in terms of {ProductPlatformName} role-based access controls.
 
 ==== Using a ServiceAccount
 
-The operator is best run using a ServiceAccount:
+The cluster operator is best run using a ServiceAccount:
 
 [source,yaml,options="nowrap"]
 .Example ServiceAccount for the Cluster Operator
 ----
-include::examples/install/cluster-operator/01-service-account.yaml[]
+include::examples/install/cluster-operator/01-ServiceAccount-strimzi-cluster-operator.yaml[]
 ----
 
-The Deployment of the operator then needs to specify this in the `serviceAccountName` of its `template.spec`:
+The Deployment of the operator then needs to specify this in its `spec.template.spec.serviceAccountName`:
 
 [source,yaml,numbered,options="nowrap",highlight='12']
 .Partial example Deployment for the Cluster Operator
 ----
-include::examples/install/cluster-operator/08-deployment.yaml[lines=1..13]
+include::examples/install/cluster-operator/05-Deployment-strimzi-cluster-operator.yaml[lines=1..13]
       # ...
 ----
 
 Note line 12, where the the `strimzi-cluster-operator` ServiceAccount is specified as the `serviceAccountName`.
 
-If the rack awareness feature is used, a Kafka init container will be started before the broker container in order to enable it.
-The application inside this container is best run using a ServiceAccount:
 
-[source,yaml,options="nowrap"]
-.Example ServiceAccount for the Kafka init container
-----
-include::examples/install/cluster-operator/04-service-account-kafka.yaml[]
-----
-
-The Cluster Operator will be in charge of configuring this ServiceAccount as `serviceAccountName` for the Kafka init container during cluster deployment.
-
-==== Defining a Role
+==== Defining ClusterRoles
 
-The Cluster Operator needs to operate using a Role that gives it access to the necessary resources.
-Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the Role.
-Cluster administrator rights are only needed for the creation of the Role.
-The Role itself does not contain any cluster administrator privileges, and the Cluster Operator will not run under the cluster admin account.
-The Role follows the "principle of least privilege" and contains only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect and Zookeeper clusters.
-The assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as StatefulSets, Deployments, Pods and ConfigMaps.
+The Cluster Operator needs to operate using some ClusterRoles that give it access to the necessary resources.
```

Comment:
Replace:

The Cluster Operator needs to operate using some ClusterRoles that give it access to the necessary resources. Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the ClusterRoles. Cluster administrator rights are only needed for the creation of the ClusterRoles. The Cluster Operator will not run under the cluster admin account. The ClusterRoles follow the "principle of least privilege" and contain only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect and Zookeeper clusters. The first set of assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as StatefulSets, Deployments, Pods and ConfigMaps.

--------
WITH:
--------

The Cluster Operator needs to operate using ClusterRoles that give it access to the necessary resources. Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the ClusterRoles. 

NOTE: Cluster administrator rights are only needed for the creation of the ClusterRoles. The Cluster Operator will not run under the cluster admin account. 

The ClusterRoles follow the "principle of least privilege" and contain only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect, and Zookeeper clusters. The first set of assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as StatefulSets, Deployments, Pods, and ConfigMaps.